components/browser/engine-gecko-nightly/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineTest.kt

```diff
@@ -68,6 +68,7 @@ import java.lang.Exception
 import org.mozilla.geckoview.WebExtension as GeckoWebExtension
 
 @RunWith(AndroidJUnit4::class)
+@Suppress("Deprecation")
```

Comment:
I would prefer if we only put that on the methods that use `registerWebExtension`. It's easier to see what's deprecated and we can address it soon.

Reply:
It's gonna have to be the methods that initializes gecko `WebExtension` constructors as well.